src/System.IO.Abstractions.TestingHelpers/MockFileSystem.cs

```diff
@@ -236,10 +236,12 @@ public void MoveDirectory(string sourcePath, string destPath)
             sourcePath = FixPath(sourcePath);
             destPath = FixPath(destPath);
 
+            var sourcePathSequence = sourcePath.Split(new[] {Path.DirectorySeparatorChar}, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Should this also split on Path.AltDirectorySeparatorChar ?
E.g. the underlying question is, what happens when you do `Directory.Move("c:/foo/bar", "c:/baz");`
If it is supported on the real filessystem the mockFilessystem should also support it.

Reply:
`FixPath` on the lines above (236 and 237) ensures only Path.DirectorySeparatorChar is in the path (Path.AltDirectorySeparatorChar is replaced).

Should it split on Path.AltDirectorySeparatorChar anyway?

Reply:
Oh, I wasn't aware of that. In that case, it is not needed.
What threw me off is that I couldn't find any tests which ensure that paths with  Path.AltDirectorySeparatorChar are actually handeled correctly, at least I couldn't find any when glancing over the code. 
Pls don't take this the wrong way, this is not a request for you to add those tests!
It is more or less a question if those tests for Path.AltDirectorySeparatorChar exist at all, e.g. at a central place or how else it is ensured that this works